sdk/PowerBI.Core/Security/GatewayCredentials.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerBI.Core.Cryptography;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.PowerBI.Core.Security
+{
+    /// <summary>
+    /// The gateway credentials used to authticate to your datasources
+    /// </summary>
+    public class GatewayCredentials
```

Comment:
Is this the credentials to authenticate with the Gateway or the Datasources behind the Gateway?